app.js

```diff
@@ -0,0 +1,299 @@
+const {
+  bootstrap,
+  helpers,
+  docker,
+} = require("@kaholo/plugin-library");
+const { promisify } = require("util");
+const exec = promisify(require("child_process").exec);
+
+const HELM_CLI_NAME = "helm";
+const HELM_IMAGE_NAME = "alpine/helm";
+
+async function install(pluginParameters) {
+  const { kubeCertificate } = pluginParameters;
+
+  let result;
+
+  await helpers.temporaryFileSentinel(
+    [kubeCertificate],
+    async (certificateFilePath) => {
```

Comment:
As you mentioned, I think we can extract this logic into a separate file, e.g.: **helm-cli.js**, every dockerified CLI plugin follows this pattern.

Reply:
Fixed